docs/app-directory/spec.md

```diff
@@ -4,7 +4,7 @@ sidebar_label: API
 title: App Directory API (next)
 ---
 
-View the [full specification](/schemas/next/app-directory) in [OpenAPI v3.0](https://www.openapis.org/) format (generated with [ReDoc](https://github.com/Redocly/redoc/)).
+View the [full specification](/schemas/next/app-directory.html) in [OpenAPI v3.0](https://www.openapis.org/) format (generated with [ReDoc](https://github.com/Redocly/redoc/)).
```

Comment:
While this code change is harmless, it also probably shouldn't be necessary. Extensions for locally-generated files will be automatically filled in.

That said, I'm surprised you don't get a warning saying that app-directory.html doesn't exist (because Docusaurus checks file references while the files are still .md)

Reply:
turned out this was necessary- at least when I did the preview.